submissions/lalalilulu/tiny-world-pre-oop/index.js

```diff
@@ -0,0 +1,98 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+
+    constructor(species, name, gender, saying, friends= [], legs) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+        this.legs = legs;
+        this.friends = friends.join(", ");
+    }
+
+    toString() {
+        return`${this.saying} I am a ${this.species}, my name is ${this.name}, my gender is ${this.gender}. I have ${this.legs} legs. My friends: ${this.friends}`;
+    }
+}
+
+class Animal extends Inhabitant{
+
+    constructor(species, name, gender, saying, friends, legs = 4) {
+        super(species, name, gender, saying, friends, legs);
+    }
+}
```

Comment:
`Animal` class looks a bit redundant. Its only purpose is to provide a default value for legs count, which is anyway explicitly assigned by its child classes.
Not a real problem in fact.